src/main/java/edu/ie3/datamodel/io/naming/EntityPersistenceNamingStrategy.java

```diff
@@ -149,13 +178,32 @@
         UUID.fromString(matcher.group("uuid")), columnScheme);
   }
 
+  /**
+   * Extracts meta information from a valid file name for a load profile time series
+   *
+   * @param fileName File name to extract information from
+   * @return Meta information form load profile time series file name
+   * @deprecated since 3.0. Use {@link #getLoadProfileTimesSeriesMetaInformation(String)} instead
+   */
+  @Deprecated(since = "3.0", forRemoval = true)
+  public edu.ie3.datamodel.io.csv.timeseries.LoadProfileTimeSeriesMetaInformation
+      extractLoadProfileTimesSeriesMetaInformation(String fileName) {
+    Matcher matcher = getLoadProfileTimeSeriesPattern().matcher(fileName);
+    if (!matcher.matches())
+      throw new IllegalArgumentException(
+          "Cannot extract meta information on load profile time series from '" + fileName + "'.");
+
+    return new edu.ie3.datamodel.io.csv.timeseries.LoadProfileTimeSeriesMetaInformation(
+        UUID.fromString(matcher.group("uuid")), matcher.group("profile"));
+  }
+
   /**
    * Extracts meta information from a valid file name for a load profile time series
    *
    * @param fileName File name to extract information from
    * @return Meta information form load profile time series file name
    */
-  public LoadProfileTimeSeriesMetaInformation extractLoadProfileTimesSeriesMetaInformation(
+  public LoadProfileTimeSeriesMetaInformation getLoadProfileTimesSeriesMetaInformation(
```

Comment:
Within the majority of the refactorings, you full omit the `get` keyword. You might consider doing it here as well!?

Reply:
ok